pyanaconda/ui/gui/spokes/network.py

```diff
@@ -41,35 +42,30 @@
 from pyanaconda.core.process_watchers import PidWatcher
 from pyanaconda.core.constants import ANACONDA_ENVIRON
 from pyanaconda.core import glib
-from pyanaconda.modules.common.constants.services import NETWORK
+from pyanaconda.modules.common.constants.services import NETWORK, NETWORK_MANAGER
 from pyanaconda.modules.common.structures.network import NetworkDeviceConfiguration
 from pyanaconda.modules.network.constants import NM_CONNECTION_TYPE_WIFI, \
     NM_CONNECTION_TYPE_ETHERNET
 
 
 from pyanaconda import network
 
-import dbus
-import dbus.service
+from dasbus.connection import SystemMessageBus
+
 # Used for ascii_letters and hexdigits constants
 import string # pylint: disable=deprecated-module
 from uuid import uuid4
 
-from dbus.mainloop.glib import DBusGMainLoop
-DBusGMainLoop(set_as_default=True)
-
 from pyanaconda.anaconda_loggers import get_module_logger
 log = get_module_logger(__name__)
 
 NM._80211ApFlags = getattr(NM, "80211ApFlags")
 NM._80211ApSecurityFlags = getattr(NM, "80211ApSecurityFlags")
 NM._80211Mode = getattr(NM, "80211Mode")
 
-NM_SERVICE = "org.freedesktop.NetworkManager"
 NM_SECRET_AGENT_GET_SECRETS_FLAG_ALLOW_INTERACTION = 0x1
-SECRET_AGENT_IFACE = 'org.freedesktop.NetworkManager.SecretAgent'
-AGENT_MANAGER_IFACE = 'org.freedesktop.NetworkManager.AgentManager'
 AGENT_MANAGER_PATH = "/org/freedesktop/NetworkManager/AgentManager"
+SECRET_AGENT_PATH = "/org/freedesktop/NetworkManager/SecretAgent"
```

Comment:
You can also create instances of `DBusObjectIdentifier` for the agent manager and the secret agent.